explore/src/main/scala/explore/proposal/ProposalEditor.scala

```diff
@@ -309,20 +314,30 @@ object ProposalEditor:
               ),
               // The second partner splits row, for maximum times - is always there
               FormStaticData(
-                value = formatHours(toHours(maxExecutionTime)),
+                value = maxExecutionPot.orSpinner(t => formatHours(toHours(t))),
                 label = maxTimeLabel,
                 id = "maxTime"
               ),
-              timeSplits(splitsMap, maxExecutionTime),
-              minimumTime(minimumPct1View.get, maxExecutionTime),
+              maxExecutionPot.renderPot(
+                valueRender = maxExecutionTime =>
+                  React.Fragment(timeSplits(splitsMap, maxExecutionTime),
+                                 minimumTime(minimumPct1View.get, maxExecutionTime)
+                  ),
+                pendingRender = React.Fragment(<.span(), <.span())
```

Comment:
Why these two spans for the pending render? An empty span by default takes up no space, so if you wanted a similar space as the render value these should probably be divs

Reply:
This was carried over from https://github.com/gemini-hlsw/explore/pull/3584. @toddburnside ?

Reply:
It is a grid layout, so the space will be determined by the grid. There just needs to be something to put in the cell to keep everything in the right cells.